### ISSUE TITLE
I am getting empty result like ActiveCaloriesBurned: []

### ISSUE THREAD

**Issue by ravi2611gupta** [matinzd/react-native-health-connect#7]
My AndroidManifest.xml file is:

<manifest xmlns:android="http://schemas.android.com/apk/res/android">

    <uses-permission android:name="android.permission.INTERNET" />

    <uses-permission android:name="android.permission.health.READ_ACTIVE_CALORIES_BURNED"/>
    <uses-permission android:name="android.permission.health.READ_HEART_RATE"/>
    <uses-permission android:name="android.permission.health.READ_SLEEP"/>
    <uses-permission android:name="android.permission.health.READ_STEPS"/>
    <uses-permission android:name="android.permission.health.READ_WEIGHT"/>
    <uses-permission android:name="android.permission.health.READ_HEIGHT"/>
    <uses-permission android:name="android.permission.health.READ_BLOOD_GLUCOSE"/>
    <uses-permission android:name="android.permission.health.READ_BLOOD_PRESSURE"/>

    <application
      android:name=".MainApplication"
      android:label="@string/app_name"
      android:icon="@mipmap/ic_launcher"
      android:roundIcon="@mipmap/ic_launcher_round"
      android:allowBackup="false"
      android:theme="@style/AppTheme">
      <activity
        android:name=".MainActivity"
        android:label="@string/app_name"
        android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|screenSize|smallestScreenSize|uiMode"
        android:launchMode="singleTask"
        android:windowSoftInputMode="adjustResize"
        android:exported="true">
        <intent-filter>
            <action android:name="android.intent.action.MAIN" />
            <category android:name="android.intent.category.LAUNCHER" />
        </intent-filter>
        <intent-filter>
          <action android:name="androidx.health.ACTION_SHOW_PERMISSIONS_RATIONALE" />
        </intent-filter>
      </activity>
    </application>
</manifest>

and my app.tsx file is: 

```tsx
import React, { useEffect } from 'react';
import type {PropsWithChildren} from 'react';
import {
  SafeAreaView,
  ScrollView,
  StatusBar,
  useColorScheme,
} from 'react-native';

import {
  Colors,
  Header,
} from 'react-native/Libraries/NewAppScreen';

import {
  initialize,
  requestPermission,
  readRecords,
} from 'react-native-health-connect';


function App(): JSX.Element {
  const isDarkMode = useColorScheme() === 'dark';


    
  const readSampleData = async () => {
    // initialize the client
    const isInitialized = await initialize();
    if(isInitialized){
       // request permissions
    const grantedPermissions = await requestPermission([
      { accessType: 'read', recordType: 'ActiveCaloriesBurned' },
      { accessType: 'read', recordType: 'BloodGlucose' },
    ]);
    
    const resultActiveCaloriesBurned = await readRecords('ActiveCaloriesBurned', {
      timeRangeFilter: {
        operator: 'between',
        startTime: '2023-01-09T12:00:00.405Z',
        endTime: new Date().toISOString(),
      },
    });
  
    const resultBloodGlucose = await readRecords('BloodGlucose', {
      timeRangeFilter: {
        operator: 'between',
        startTime: '2023-01-09T12:00:00.405Z',
        endTime: new Date().toISOString(),
      },
    });
  
    console.log("Result is: ", resultActiveCaloriesBurned, resultBloodGlucose);
    
  
    }else{
      console.log("Sorry! something went wrong!");
    }
  
  };
  
  useEffect(()=>{
    readSampleData();
  }, [])
  

  const backgroundStyle = {
    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
  };

  return (
    <SafeAreaView style={backgroundStyle}>
      <StatusBar
        barStyle={isDarkMode ? 'light-content' : 'dark-content'}
        backgroundColor={backgroundStyle.backgroundColor}
      />
      <ScrollView
        contentInsetAdjustmentBehavior="automatic"
        style={backgroundStyle}>
        <Header />
      
      </ScrollView>
    </SafeAreaView>
  );
}


export default App;
```

**Comment by matinzd** [comment]
I also faced this issue and I investigated it around a week ago. After going deeper and more debugging I could see that the values are being passed properly to Health Connect SDK and although everything is fine, Health Connect is not returning anything. It seems the issue started happening after releasing alpha10 from google. 
I am in contact with them to see what has happened during alpha10 migration.
